travis/scripts/05-run.sh

```diff
@@ -47,7 +47,11 @@ launchCurlOrProtractor() {
         if [[ -f "gulpfile.js" ]]; then
             gulp itest --no-notification
         elif [[ -f "tsconfig.json" ]]; then
-            yarn e2e
+            pushd $HOME
```

Comment:
@pascalgrimaud WDYT? is this ok?

Reply:
Sorry, my comment has been deleted. So I write again ! 

@deepu105. Systematically there is a bug with with `Error: Could not find update-config.json. Run 'webdriver-manager update' to download binaries.`. For wich one with nearer success, it's that too (see https://travis-ci.org/jhipster/generator-jhipster/builds/360247194).

So I've followed what @gmarziou said with his reference to [angular/webdriver-manager#269 (comment)](https://github.com/angular/webdriver-manager/issues/269#issuecomment-371623733).

Fail again, because chromedriver seems to not be downloaded
```
Saving to: ‘chromedriver_linux64’
100%[======================================>] 3,719,424   --.-K/s   in 0.02s   
2018-03-31 01:09:11 (161 MB/s) - ‘chromedriver_linux64’ saved [3719424/3719424]
unzip:  cannot find or open chromedriver_linux64.zip, chromedriver_linux64.zip.zip or chromedriver_linux64.zip.ZIP.
```
(see https://travis-ci.org/jhipster/generator-jhipster/builds/360267542). 
Have we write access in $HOME ? Maybe try without `push $HOME && popd` ? See below…